html-css-js/.hintrc

```diff
@@ -0,0 +1,17 @@
+{
+  "connector": {
+    "name": "local",
+    "options": {
+      "pattern": ["**", "!.git/**", "!node_modules/**"]
+    }
+  },
+  "extends": ["development"],
+  "formatters": ["stylish"],
+  "hints": [
+    "button-type",
+    "disown-opener",
+    "html-checker",
+    "meta-charset-utf-8",
+    "meta-viewport"
```

Comment:
Let's add the new hint too

Reply:
Will do! 
I completely forgot about the readme, I'll add it as well